### ISSUE TITLE
Problem with comments in python file

### ISSUE THREAD

**Issue by ThunderHeavyIndustries** [onaclov2000/Programming_Cognate#1]
I think syntax would be the greater challenge than simply translating words ;p

**Comment by onaclov2000** [comment]
I agree, but conceptually, it would be similar to languages in that we have similar constructs across them, and the words are the first step, as we have seen with languages, crossing from english to spanish for example a word for word translation results in a "script error" in the other language, so for example understanding contextually that say def is usually preceeding a function to and that a function is usually preceeded by the return type, would allow some kind of translation between say python and c. I apologize I can't think of any other really great examples, but I'm also thinking of things like how there are male/female and proper vs regular usages of a word in spanish, for the same single word in english,  so being able to recognize that as a core to the translation might be helpful.

just throwing ideas out there:)
